param/parameterized.py

```diff
@@ -209,6 +209,25 @@ def _f(*args, **kwargs):
     return _f
 
 
+def disable_instance_params(cls):
```

Comment:
Maybe `no_instance_params`?  Disabling sounds like there could be instance parameter objects they just would be non-functional, where `no` seems more like a declaration that there are not and should not be any instance params.